modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -163,10 +192,11 @@ export default {
         apos.bus.$emit('busy', false);
       }
     },
-    populateSchema () {
+    buildEmptyDoc () {
       this.schema.forEach(field => {
-        this.doc.data[field.name] = {};
```

Comment:
This was a bug, but we didn't know because the schema jawn was being generated before it was done building this.